Tests/GitCheckTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Xunit;
+using FluentAssertions;
+using YoCode;
+using System.Collections.Generic;
+
+namespace YoCode_XUnit
+{
+    public class GitCheckTests
+    {
+        string testOutput,testString,testLastAuthor;
+        GitCheck gc;
+        List<string> testDomainList;
+        public GitCheckTests()
+        {
+            testString = "commit fd891f3dbcf0fd935814b56f87b2e0f768fe5bef (HEAD -> i8-gitcheck)\n" +
+                "\nAuthor: matas.zilaitis < matas.zilaitis@gmail.com > " +
+                "\n Date:   Thu Jul 5 16:57:15 2018 + 0100 " +
+                "\nChanged the class logic a bit thanks to Mike";
+            testLastAuthor = "Author: matas.zilaitis < matas.zilaitis@gmail.com > ";
+            gc = new GitCheck();
+
+            testDomainList = new List<string>();
+            testDomainList.Add("@nonlinear.com");
+            testDomainList.Add("@nonlinear.com");
+
+        }
+
+
+        [Fact]
+        public void Test_GetLastAuthor()
+        {
+            testLastAuthor.Should().BeEquivalentTo(gc.getLastAuthor(testString));
+            
+        }
+
+
+        [Fact]
+        public void Test_GitHasBeenUsed()
+        {
+            bool testBool = true;
+            testBool.Should().Be(gc.GitHasBeenUsed(testLastAuthor,gc.getHostDomains()));
+            
+        }
+
+        [Fact] 
+        public void Test_ContainsAny()
+        {
+            false.Should().Be(GitCheck.ContainsAny(testLastAuthor,gc.getHostDomains()));
```

Comment:
I think all of these asserts would read better if they were written the other way round. It's more normal to write them in the order `actual.Should().Be(expected)`. Yours are `expected.Should().Be(actual)`. The reason for the difference is that examples like this one would be a bit nicer to read. If you change the order it would say `GitCheck.ContainsAny(testLastAuthor,gc.getHostDomains()).Should().Be(false)`. That has two benefits I think - it reads a bit better ("GitCheck ContainsAny should be false" is nicer than "false should be GitCheck ContainsAny") and it also makes it easier to see what is being tested because it's at the start of the line of code rather than the end.

Reply:
fixed